api/src/test/resources/underlays/nautical.pbtext

```diff
@@ -138,6 +142,17 @@ datasets: {
       data_type: STRING
     }
   }
+  tables: {
```

Comment:
So indexes will generally be tables, but not entities themselves - correct?

Reply:
Yep, indexes are tables to support operations on entities, but they're not exposed as separate entities.
e.g. we also want a free text search index. that will be backed initially by a postgres table and be related to an entity, but it's not a distinct entity in itself.